packages/menu/stories/menu.stories.ts

```diff
@@ -431,3 +431,24 @@ export const menuWithValueSlots = (): TemplateResult => {
         </sp-popover>
     `;
 };
+
+headersAndIcons.storyName = 'Dynamic MenuItems';
+
+export const dynamicRemoval = (): TemplateResult => {
```

Comment:
Dynamic removal test is timing out in Chromium, so I've been using this story to try to debug that. 

Note: This story works.